repo-checkout/action.yml

```diff
@@ -4,12 +4,12 @@
 
 name: 'Repo Checkout'
 description: |
-  Does sel4test test runs from previously built images for all hardware test platforms.
+  Checks out repositories based on a manifest file.
 author: Gerwin Klein <gerwin.klein@proofcraft.systems>
 
 inputs:
   manifest_repo:
-    description: Manifest repostory (e.g. 'sel4test-manifest')
+    description: Manifest repostory (e.g. 'sel4test-manifest', 'sel4bench-manifest')
```

Comment:
Why add sel4bench specifically? There are many more that could be listed here.

Reply:
Some reasons:
- two references are better than one
- both sel4test and sel4bench are a kind of minimum reference for all the rest
... otherwise, nothing specific. Main practical reason might be, that I'm currently working on the sel4bench repo.

Reply:
> - two references are better than one

Strongly disagree with that one :-) Useful values for references are 0, 1, or all. 2 is random.

> both sel4test and sel4bench are a kind of minimum reference for all the rest
> ... otherwise, nothing specific. Main practical reason might be, that I'm currently working on the sel4bench repo.

It's fine to merge it, it's not wrong, I was just wondering, because sel4bench is probably the least used manifest.